sas9.4-viya/sas_clients.ps1

```diff
@@ -23,10 +23,13 @@ Set-PSDebug -Trace 1;
 $logdir = "C:\saslog"
 $mid_fqdn= "${app_name}${mid_name}.${domain_name}"
 New-Item -Path $logdir -ItemType directory
-Invoke-WebRequest -Uri ${artifact_loc}properties/clients_install.properties -OutFile ${logdir}\clients_install.properties
-ExitWithCode
-Invoke-WebRequest -Uri ${artifact_loc}properties/plan.xml -OutFile ${logdir}\plan.xml
+
+Invoke-WebRequest -Uri https://github.com/corecompete/sasinstalls/archive/main.zip -OutFile sasinstall.zip
```

Comment:
this must be included in the sample, cannot be downloaded from an external source

Reply:
changed the code logic accordingly.